src/ImageSharp/Colors/ColorVector.Definitions.cs

```diff
@@ -0,0 +1,728 @@
+﻿// <copyright file="ColorVector.Definitions.cs" company="James Jackson-South">
```

Comment:
Sweet. I'll be duplicating the `Color.Transforms` class also eventually. Just wanna get this in first.